### ISSUE TITLE
Remove the news category sidebar (Following up on #851)

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#911]
@brockfanning converting to issue so we can discuss DAC's response:

> In general, I would not say that information was supplementary if the lack of that information does not stop the user achieving their main goal. We would expect all users to be able to access the same information and functionality.

> I would say that supplementary information is perhaps an additional visual cue for sighted users that does not provide any information or functionality but is just decorative.

> This navigation region isn’t decorative as it provides functionality. This is just a usability comment, so to achieve AA standard it does not need to be met. It is just in the report to make you aware that blind users will not be able to access that information as easily as a sighted user who can immediately see this information on the page.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning @phillipgardiner what are your thoughts on DAC's response?

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning one of the things that have come up previously in UK side of things is removing the categories part of the page. We also had a request from Kyrgyzstan recently where they asked to also remove it.

I think it is just the UK and Kyrgyzstan who use the News page, so maybe we could/should remove the categories section?

Have also discussed with Ann and Phill offline and they think so too

**Comment by brockfanning** [comment]
I don't have any objections to removing it.
